packages/shared/src/components/CommentFeed.tsx

```diff
@@ -77,22 +82,45 @@ export default function CommentFeed<T>({
   }
 
   if (showEmptyScreen && emptyScreen) {
-    return <>{emptyScreen}</>;
+    return (
+      <>
+        {isNewMobileLayout && isMainFeed && (
+          <FeedSelector className="self-start py-10 px-4" />
+        )}
```

Comment:
Given how the UI is set up, you wouldn't be able to navigate to popular or upvoted without going back through the nav tabs, it would also make the UX weird as the dropdown would disappear when you navigate to this feed:

<img width="775" alt="Screenshot 2024-03-27 at 20 09 49" src="https://github.com/dailydotdev/apps/assets/1225100/320c9ecf-a7ae-496e-a8a6-f2a78da993f3">

Reply:
Not disappear, but be a placeholder here.
It will avoid loading all the extra data that you can't use yet.

Reply:
Oh wait sorry got you :)